.circleci/config.yml

```diff
@@ -47,8 +48,42 @@ commands:
             # "!" negates return code. exit nonzero if any of these deps are found
             ! pip freeze | grep -E "xgboost|catboost|lightgbm|plotly|ipywidgets"
             exit $?
+  build_pkg:
+    steps:
+      - run: |
+            git clone https://github.com/conda-forge/evalml-core-feedstock
+            source test_python/bin/activate
+            mkdir evalml-core-feedstock/evalml
+            cp -r `ls -A | grep -v "evalml-core-feedstock"` ./evalml-core-feedstock/evalml/
+            python conda_config.py "$(python setup.py --version)"
+            cd evalml-core-feedstock
+            echo "$DOCKER_HUB_PASS" | docker login -u psalter --password-stdin
```

Comment:
We have manually check out this container and run `docker run` ourselves as opposed to running `run_docker_build.sh` in the feedstock repo because circle-ci doesn't let you mount volumes to the docker image. I think this is ok because we're not straying far from the contents of that script. @ryan-s and I discussed ways of tightening this up in the future too!

Reply:
@ryan-s @freddyaboulton is this env var `DOCKER_HUB_PASS` set by circleci? Did you have to enter it?

Reply:
@dsherry I entered it in circleci. Not sure if we have a vault somewhere to share the credentials?